android/src/test/java/com/reactnativecommunity/asyncstorage/next/StorageTest.kt

```diff
@@ -0,0 +1,141 @@
+package com.reactnativecommunity.asyncstorage.next
+
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import org.json.JSONObject
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.random.Random
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+class AsyncStorageAccessTest {
```

Comment:
Should we also add tests for migration?

Reply:
I'd say that it could be valuable if we would do some manipulation of the data while migrating. Right now, we simply copy data from older storage to new one as it is